articles/libraries/_includes/_configure_custom_domain.md

```diff
@@ -0,0 +1,29 @@
+### Configure your custom domain
+
+1. Ensure that your application updates are made prior to setting up custom domains in your application, as the older versions of ${library} will not work when configured with a custom domain.
+
+2. Set up your [custom domain](/custom-domains) in the Auth0 [Dashboard](${manage_url}/#/tenant).
+
+3. Use the custom domain when instantiating Lock.
+
+```
+var lock = new Auth0Lock('your-client-id', 'login.northwind.com', options);
+```
+
+4. Set the `configurationBaseUrl` option to `https://cdn.auth0.com`.
+
+```
+var options = {
+  configurationBaseUrl: 'https://cdn.auth0.com'
+}
+```
+
+::: note
+The CDN URL varies by region. For regions outside of the US, use `https://cdn.{region}.auth0.com`.
```

Comment:
maybe add parenthesis at the end that clarifies the regions? sth like `use eu from Europe, au for Australia`